paperqa/litqa.py

```diff
@@ -77,8 +98,8 @@ class LitQAEvaluation(IntEnum):
     """Possible evaluation results for a LitQA question."""
 
     CORRECT = 0
-    INCORRECT = 1
-    UNSURE = 2
+    INCORRECT = 2
+    UNSURE = 1
```

Comment:
Why rely on indexing instead of e.g. using a mapping?

Reply:
Nice suggestion, done 👌